includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1483,6 +1554,7 @@ protected function build_display_items( $itemized_display_items = false ) {
 				$quantity_label = 1 < $cart_item['quantity'] ? ' (x' . $cart_item['quantity'] . ')' : '';
 
 				$product_name = $cart_item['data']->get_name();
+				// $product_name = $product->get_name(); // which one should we use?
```

Comment:
> which one should we use?

I think it's fine to use `$cart_item['data']->get_name()`. Otherwise where would we get this variable `$product` from? We're iterating each cart item.

Reply:
Another comment for me. I forgot to add the "TODO" prefix. Also, you are correct.